### ISSUE TITLE
Unable to auto-start and also crashing after a few days running

### ISSUE THREAD

**Issue by acurcione** [gfjardim/docker-containers#61]
I posted this to the unRAID support thread, but haven't heard from anyone. Thought I'd try this.

My Crashplan docker continues to crash randomly, about a week or so, and I have no idea what it's complaining about or why it's crashing. This is what pops in the log when it happens:

*** Shutting down runit daemon (PID 22)...
XIO: fatal IO error 11 (Resource temporarily unavailable) on X server ":1"
after 2120 requests (2120 known processed) with 0 events remaining.
./run: line 20: 39 Killed $JAVACOMMON $SRV_JAVA_OPTS -classpath "$TARGETDIR/lib/com.backup42.desktop.jar:$TARGETDIR/lang" com.backup42.service.CPService > /config/log/engine_output.log 2> /config/log/engine_error.log

Unfortunately if I go and look for that engine_error log it mentions it's completely blank so this is all I have unless someone can point me in another direction.

I also have a problem with this docker and autostart - as in it can't for whatever reason. Also asked in the support thread and got nothing. This is that error, in case somehow it's all related.

ERROR: openbox-xdg-autostart requires PyXDG to be installed 

Not really sure why it's complaining about that. It's not like I have an option to install it.

**Comment by jlesage** [comment]
The autostart issue has also been reported in the beginning of the year on unRAID forum, without answer...

https://forums.lime-technology.com/topic/32220-container-crashplan-amp-crashplan-desktop/?page=46#comment-522851

**Comment by acurcione** [comment]
Yep. I know that as well. I was hoping posting here would get the developers attention on this finally.
